corehq/apps/commtrack/views.py

```diff
@@ -111,6 +111,9 @@ def post(self, request, *args, **kwargs):
                 if data.get('stock_' + field):
                     setattr(self.commtrack_settings.stock_levels_config, field,
                             data['stock_' + field])
+            # touch the location_types to update their stock levels config
+            for loc_type in LocationType.objects.filter(domain=self.domain).all():
```

Comment:
rails has a nice utility for this called `touch`.  that does exactly what your comment suggests. maybe it'll be added in django 2.0